### ISSUE TITLE
HTTP Error 404: Not Found

### ISSUE THREAD

**Issue by Xtrau505** [klept0/MS-Rewards-Farmer#72]
### Before submitting a bug report...

- [x] This bug wasn't already reported.
(I have checked every bug report on GitHub)

### Title

- [X] The title is no longer "[BUG] Title" and I edited it with the right error name.

### Describe the bug

I cant seem to use undetected_chromedriver and I cant find out why since this only started happening today.

### Copy and paste your error

2024-02-15 08:01:22,241 [ERROR] HTTPError: HTTP Error 404: Not Found
Traceback (most recent call last):
  File "C:\Users\james\farmer2\MS-Rewards-Farmer\main.py", line 47, in main
    earned_points = executeBot(currentAccount, notifier, args)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\farmer2\MS-Rewards-Farmer\main.py", line 221, in executeBot
    with Browser(mobile=False, account=currentAccount, args=args) as desktopBrowser:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\farmer2\MS-Rewards-Farmer\src\browser.py", line 41, in __init__
    self.webdriver = self.browserSetup()
                     ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\farmer2\MS-Rewards-Farmer\src\browser.py", line 86, in browserSetup
    driver = webdriver.Chrome(
             ^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\site-packages\seleniumwire\undetected_chromedriver\webdriver.py", line 61, in __init__
    super().__init__(*args, **kwargs)
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\site-packages\undetected_chromedriver\__init__.py", line 258, in __init__
    self.patcher.auto()
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\site-packages\undetected_chromedriver\patcher.py", line 178, in auto
    self.unzip_package(self.fetch_package())
                       ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\site-packages\undetected_chromedriver\patcher.py", line 287, in fetch_package
    return urlretrieve(download_url)[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 240, in urlretrieve
    with contextlib.closing(urlopen(url, data)) as fp:
                            ^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 215, in urlopen
    return opener.open(url, data, timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 521, in open
    response = meth(req, response)
               ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 630, in http_response
    response = self.parent.error(
               ^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 559, in error
    return self._call_chain(*args)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 492, in _call_chain
    result = func(*args)
             ^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 639, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

### Screenshots

the window will not open.

### Value of dashboard variable

https://gist.github.com/Xtrau505/b2b21785ab2b6deb5f3c2a803b4e6250

**Comment by MacMarde** [comment]
I have the exact same error

**Comment by MacMarde** [comment]
@Xtrau505 
Have you found a solution?

**Comment by agustinscaz** [comment]
Same error here... installing the requirements does not solve the problem

**Comment by MacMarde** [comment]
I have started testing the bot today. It was working a short time. 
If I remember correctly it stopped working after I tried to change the language from the command prompt.
But of course it could have been something different.
Anyway I deleted the whole project, extracted from the ZIP eveything again, restarted my PC but still it just stopped working.

**Comment by klept0** [comment]
This is an issue with [undetected chromedriver ](https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/1743)- being that people do not look into the issue themselves or try to troubleshoot I will see what I can do when I have free time unless someone else resolves it prior

**Comment by dudeua** [comment]
In layman's terms on windows:
Navigate to the undetected_chromedriver package
Default should be:  `Users\User\Appdata\Local\Programs\Python\PythonVERSION\Lib\site-packages\undetected_chromedriver`
Edit `patcher.py`
Find
`download_url = "https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/%s/%s/%s"`
change to
`download_url = "https://storage.googleapis.com/chrome-for-testing-public/%s/%s/%s"`
